### PR TITLE
Add resnext, fix cifar data augmentation

### PR DESCRIPTION
**Pull request by hetong007**
ResNext training will be added later.

So far the cifar augmentation has called `numpy.pad`, since we only have a limited `nd.pad` as mentioned here: https://github.com/apache/incubator-mxnet/issues/10818

### CODE REVIEW

**Comment by mli** [comment]
Job PR-90-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-90/3/index.html

**Comment by mli** [comment]
Job PR-90-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-90/4/index.html

**Comment by mli** [comment]
Job PR-90-6 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-90/6/index.html

**Comment by mli** [comment]
Job PR-90-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-90/7/index.html

File: gluoncv/model_zoo/model_store.py [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
-    ('5369ff00708900aa34c85088ac2c15fd42f0b243', 'cifar_wideresnet40_8'),
+    ('121e1579d811b091940b3b1fa033e1f0d1dca40f', 'cifar_resnet20_v1'),
+    ('4f2d18804c94f2d283b8b45256d048bd3d6dd479', 'cifar_resnet20_v2'),
+    ('2fb251e60babdceb103e9659b3baa0dea20a14d7', 'cifar_resnet56_v1'),
+    ('0a3e74104ec7bcfffefe2d9d5cc1f8e74311ec51', 'cifar_resnet56_v2'),
+    ('a0e1f860475bf5369f6da07e0c2e03a4ae9cff9c', 'cifar_resnet110_v1'),
+    ('bf160f8b3cb3884a1ea871739f3c8e151e114159', 'cifar_resnet110_v2'),
+    ('7c07b5ba6e850f9c37ca1e57c0a2e529455cc2e4', 'cifar_wideresnet16_10'),
+    ('4a3466aadd4c3ddbcb968bca862d0e59d6f15ec1', 'cifar_wideresnet28_10'),
+    ('085ca2afabbe0ddfe87d0edc5408bcfcfbffd414', 'cifar_wideresnet40_8'),
```
Comment (zhanghang1989) [inline gluoncv/model_zoo/model_store.py]: Is the performance different than reported in the model zoo docs?
http://gluon-cv.mxnet.io/model_zoo/index.html

Comment (hetong007) [reply gluoncv/model_zoo/model_store.py]: They all achieve better results. The doc's also updated, please check the pre-built temporary page.

**Comment by zhanghang1989** [comment]
Could you please also add the changes to the release note
https://github.com/dmlc/gluon-cv/releases

File: gluoncv/model_zoo/model_store.py (hunk 1 above)
Comment (zhanghang1989) [reply gluoncv/model_zoo/model_store.py]: I see it now :)

**Pull request merged**